command.go

```diff
@@ -93,13 +94,13 @@ type Command struct {
 
 var (
 	// ErrNotCallable signals a command that cannot be called.
-	ErrNotCallable = ClientError("This command can't be called directly. Try one of its subcommands.")
+	ErrNotCallable = ClientError("this command can't be called directly. Try one of its subcommands.")
```

Comment:
I wouldn't fix this unless we get rid of all caps: "this command can't be called directly; try one of its subcommands"